huaweicloud/services/rds/resource_huaweicloud_rds_instance.go

```diff
@@ -126,6 +126,16 @@ func ResourceRdsInstance() *schema.Resource {
 							Computed: true,
 							ForceNew: true,
 						},
+						"limit_size": {
+							Type:         schema.TypeInt,
+							Optional:     true,
+							RequiredWith: []string{"volume.0.trigger_threshold"},
+						},
+						"trigger_threshold": {
+							Type:         schema.TypeInt,
+							Optional:     true,
+							RequiredWith: []string{"volume.0.limit_size"},
```

Comment:
just keep one RequiredWith will be fine

Reply:
A required with B:
if A configured and B is omitted, the provider will remind B need configure.
but if B configured and A is omitted, the provider will think it is ok.
A required with B && B required with A:
whatever A is omitted or B is omitted, the provider will remind the required parameter should be configured.